### PR TITLE
Tweet changelog entry

### PR DESCRIPTION
**Pull request by kntsoriano**
https://github.com/exercism/wip/issues/35

### CODE REVIEW

File: test/models/changelog_entry_tweet_formatter_test.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+require "test_helper"
+
+class ChangelogEntryTweetFormatterTest < ActiveSupport::TestCase
+  test "returns empty string if tweet_copy is empty" do
+    entry = create(:changelog_entry, tweet_copy: nil)
+
+    assert_equal "", ChangelogEntryTweetFormatter.format(entry)
```
Comment (iHiD) [inline test/models/changelog_entry_tweet_formatter_test.rb]: Should this return if the tweet_copy is empty, to help ensure we never tweet it empty?

File: app/models/twitter_account.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    TweetJob.perform_later(
+      consumer_key: consumer_key,
+      consumer_secret: consumer_secret,
+      access_token: access_token,
+      access_token_secret: access_token_secret,
+      copy: copy
+    )
+  end
+
+  def ==(other)
```
Comment (iHiD) [inline app/models/twitter_account.rb]: Would it be better to also pass the `slug` in and then `==` based on the slug?

Comment (kntsoriano) [reply app/models/twitter_account.rb]: @iHiD Ah yes. Good idea :)

Comment (iHiD) [reply app/models/twitter_account.rb]: Whoop! :)

**Pull request merged**